test/e2e/framework/node/resource.go

```diff
@@ -46,6 +45,21 @@ const (
 	sshPort = "22"
 )
 
+var (
+	// unreachableTaintTemplate is the taint for when a node becomes unreachable.
+	unreachableTaintTemplate = &v1.Taint{
```

Comment:
Can we add a comment referencing where this taint is originally from?

Reply:
Sure!